packages/tupaia-web/src/features/Dashboard/ExportDashboard/Preview.tsx

```diff
@@ -0,0 +1,232 @@
+  padding: 1rem;
+`;
+
+const Legend = styled.legend`
+  color: ${({ theme }) => theme.palette.text.primary};
+  font-size: 1rem;
+  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+`;
+
+const Checkbox = styled(BaseCheckbox)`
+  margin: 0.5rem 0 0 1rem;
+  .MuiButtonBase-root {
+    padding: 0;
+    margin-right: 0.5rem;
+  }
+  label {
+    padding: 0.5rem 0 0 0.5rem;
+  }
+`;
+
+const PreviewPanelContainer = styled.div`
+  height: 100%;
+  width: 38%;
+  display: flex;
+  flex-direction: column;
+`;
+
+const ExportSettingsInstructionsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  padding-bottom: 0.7rem;
+`;
+
+const PreviewHeaderContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 0.3rem;
+`;
+
+const PreviewPagination = styled(Pagination)`
+  .MuiPaginationItem-page {
+    font-size: 0.7rem;
+  }
+`;
+
+const PreviewContainer = styled.div`
+  display: flex;
+  background-color: white;
+  height: 30rem;
+  min-width: 20rem;
+  overflow: auto;
+`;
+
+const Instructions = styled(Typography)`
+  color: ${({ theme }) => theme.palette.text.primary};
+  font-size: 0.875rem;
+  line-height: 1.4;
+`;
+
+const PreviewTitle = styled(Typography)`
+  color: ${({ theme }) => theme.palette.text.primary};
+  font-weight: bold;
+  font-size: 0.875rem;
+  line-height: 1.4;
+`;
+
+interface ExportDashboardProps {
+  onClose: () => void;
+  selectedDashboardItems: string[];
+}
+
+export const Preview = ({ onClose, selectedDashboardItems = [] }: ExportDashboardProps) => {
+  const { projectCode, entityCode, dashboardName } = useParams();
+  const { data: project } = useProject(projectCode);
+  const { data: entity } = useEntity(projectCode, entityCode);
+  const [page, setPage] = useState(1);
+  const onPageChange = (_: unknown, newPage: number) => setPage(newPage);
+  const visualisationToPreview = selectedDashboardItems[page - 1];
+
+  const handleExportSuccess = (data: Blob) => {
+    downloadJs(data, `${exportFileName}.pdf`);
+  };
+
+  const { mutate: requestPdfExport, error, isLoading, reset } = useExportDashboard({
+    onSuccess: handleExportSuccess,
+  });
+
+  const exportFileName = `${project?.name}-${entity?.name}-${dashboardName}-dashboard-export`;
+
+  const handleExport = () =>
+    requestPdfExport({
+      projectCode,
+      entityCode,
+      dashboardName,
+      selectedDashboardItems,
+    });
+
+  return (
+    <LoadingContainer
+      heading="Exporting charts to PDF"
+      isLoading={isLoading}
+      errorMessage={error?.message}
+      onReset={reset}
+    >
+      <PrimaryContext>
+        <Container>
+          <ExportSettingsContainer>
+            <ExportSettingsInstructionsContainer>
+              <Instructions>Edit export settings and click 'Download'.</Instructions>
+            </ExportSettingsInstructionsContainer>
+            <ExportSetting>
+              <FormGroup>
```

Comment:
One small thing - can we wrap this in a `fieldset` to make sure it meets accessibility best practices?

Reply:
Were you thinking a `fieldset` to make it visually clear which settings are grouped together?

In the final designs we're planning on having dividers between the different options which might achieve the same purpose
![image](https://github.com/beyondessential/tupaia/assets/59544282/0f3d53fc-516e-48d1-bc32-c219107603e1)

Or is there something specific about `fieldset` when using a screenreader or something?

Reply:
Fieldsets can be 'invisible', but they're more for accessibility reasons - it tells assistive technologies that a group of inputs are a group, instead of multiple individual inputs. You can totally restyle the fieldset to not add in those default styles if you already have them as part of another component or something

Reply:
I'll add the invisible fieldset I reckon :+1: